packages/core/admin/admin/src/components/GuidedTour/Stepper/StepNumber.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { pxToRem } from '@strapi/helper-plugin';
+import { Flex } from '@strapi/design-system/Flex';
+import { Typography } from '@strapi/design-system/Typography';
+import { Icon } from '@strapi/design-system/Icon';
+import Check from '@strapi/icons/Check';
+import { IS_DONE, IS_ACTIVE, IS_NOT_DONE } from '../constants';
+
+const StepNumber = ({ type, number }) => {
+  if (type === IS_DONE) {
+    return (
+      <Flex
+        background="primary600"
+        padding={2}
+        borderRadius="50%"
+        width={pxToRem(30)}
+        height={pxToRem(30)}
+        justifyContent="center"
+      >
+        <Icon as={Check} aria-hidden width="16px" color="neutral0" />
```

Comment:
The parent container's width & height uses rem unit but not the Icon, is it wanted?

Reply:
it's not, my bad, thank you for catching it!